libvcx/src/api/return_types_u32.rs

```diff
@@ -111,6 +111,38 @@ impl Return_U32_U32 {
     }
 }
 
+#[allow(non_camel_case_types)]
+pub struct Return_U32_I32 {
+    pub command_handle: CommandHandle,
+    receiver: Receiver<(u32, i32)>,
+}
+
+impl Return_U32_I32 {
```

Comment:
Where is this  struct used?

Reply:
All of these return type structs exists basically just for testing FFI. This is used in  `test_open_wallet` because it's expressing the return value of `vcx_open_main_wallet`